src/redux/rockets/rocketsSlice.js

```diff
@@ -0,0 +1,40 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+export const fetchRockets = createAsyncThunk('rockets/fetchRockets', async () => {
+  try {
+    const response = await axios.get('https://api.spacexdata.com/v4/rockets');
+    return response.data;
+  } catch (error) {
+    throw Error('Failed to fetch rocket data');
+  }
+});
+
+const initialState = {
+  rockets: [],
+  status: 'idle',
```

Comment:
You could use boolean values for status, to make conditional rendering later, for example for missions I did:

![image](https://github.com/victormc13/Space-Travelers-Hub/assets/110580299/41c48dac-c9b2-4a3f-a68d-1ed9eb442732)